templates/interstitial.html

```diff
@@ -1,14 +1,14 @@
 {% extends 'layouts/_questionnaire.html' %}
 
+{% import 'macros/helpers.html' as helpers %}
+
 {% set save_on_signout = true %}
 
 {% set continue_button_text = _("Continue") %}
 
 {% block form_content %}
   {% set interstitial_instruction = format_paragraphs(block.content.instruction) if block.content.instruction else None %}
-  {% if block.content.title %}
-    <h1>{{ block.content.title }}</h1>
-  {% endif %}
+  {{ helpers.interviewer_note(block.content.interviewer_only, block.content.title, block.type) }}
```

Comment:
This feels like it should be only called if interviewer note formatting is required. It would help simplify the macro to move the interviewer_only and interstitial checks to this partial.

Reply:
I had to keep block type check in macro but otherwise it's all done(I hope)👍